compose/service/values/shared.go

```diff
@@ -0,0 +1,37 @@
+package values
+
+import (
+	"regexp"
+	"time"
+)
+
+const (
+	strBoolTrue  = "1"
+	strBoolFalse = "0"
+
+	datetimeInputFormatDate = "2006-01-02"
+	datetimeInputFormatTime = "15:04:05"
```

Comment:
should we not handle TZ?

Reply:
we do (in sanitizer), but these are internal datetime formats